### PR TITLE
BZ2032484: added an admonition

### PR DESCRIPTION
**Pull request by Amrita42**
- Applies to 4.9+
- [Bugzilla](https://bugzilla.redhat.com/show_bug.cgi?id=2032484)
- [Preview](https://deploy-preview-40028--osdocs.netlify.app/openshift-enterprise/latest/networking/multiple_networks/configuring-additional-network)
- @phoracek @zhaozhanqi  please review and ack.

### CODE REVIEW

File: networking/multiple_networks/configuring-additional-network.adoc [hunk 1]
```diff
@@ -26,7 +26,12 @@ For an additional network, IP addresses are provisioned through an IP Address Ma
 [id="{context}_configuration-additional-network-attachment"]
 == Configuration for an additional network attachment
 
-An additional network is configured via the `NetworkAttachmentDefinition` API in the `k8s.cni.cncf.io` API group. The configuration for the API is described in the following table:
+An additional network is configured via the `NetworkAttachmentDefinition` API in the `k8s.cni.cncf.io` API group.
+[IMPORTANT]
+====
+Do not store any sensitive information or a secret in the `NetworkAttachmentDefinition` object because this information is accessible by the unprivileged user.  
+====
+The configuration for the API is described in the following table:
```
Comment (ctauchen) [inline networking/multiple_networks/configuring-additional-network.adoc:34]: You might add a space before this line.

File: networking/multiple_networks/configuring-additional-network.adoc [hunk 2]
```diff
@@ -26,7 +26,12 @@ For an additional network, IP addresses are provisioned through an IP Address Ma
 [id="{context}_configuration-additional-network-attachment"]
 == Configuration for an additional network attachment
 
-An additional network is configured via the `NetworkAttachmentDefinition` API in the `k8s.cni.cncf.io` API group. The configuration for the API is described in the following table:
+An additional network is configured via the `NetworkAttachmentDefinition` API in the `k8s.cni.cncf.io` API group.
+[IMPORTANT]
+====
+Do not store any sensitive information or a secret in the `NetworkAttachmentDefinition` object because this information is accessible by the unprivileged user.
```
Comment (ctauchen) [inline networking/multiple_networks/configuring-additional-network.adoc:32]: I don't know that we'd call this an 'unprivileged' user. From the BZ, it looks like access to this information was expanded from only cluster admin to both cluster and project admins. But there are other roles that wouldn't have access.

**Review by ctauchen — COMMENTED** [review]
Thanks for this -- comments below.

File: networking/multiple_networks/configuring-additional-network.adoc (hunk 2 above)
Comment (zhaozhanqi) [reply networking/multiple_networks/configuring-additional-network.adoc:32]: it should be project admin user

File: networking/multiple_networks/configuring-additional-network.adoc [hunk 3]
```diff
@@ -26,7 +26,13 @@ For an additional network, IP addresses are provisioned through an IP Address Ma
 [id="{context}_configuration-additional-network-attachment"]
 == Configuration for an additional network attachment
 
-An additional network is configured via the `NetworkAttachmentDefinition` API in the `k8s.cni.cncf.io` API group. The configuration for the API is described in the following table:
+An additional network is configured via the `NetworkAttachmentDefinition` API in the `k8s.cni.cncf.io` API group.
+[IMPORTANT]
+====
+Do not store any sensitive information or a secret in the `NetworkAttachmentDefinition` object because this information is accessible by the project administration user.
```
Comment (ctauchen) [inline networking/multiple_networks/configuring-additional-network.adoc:32]: I'm confused about the logic here -- it suggests that, by definition, project admins are not to be trusted with 'sensitive information'. Surely that's a call that the cluster admins should make?

I wonder if we should be saying something more like this:

"The `NetworkAttachmentDefinition` object can be accessed by both project and cluster administrators. You should not store sensitive information in this object if that information cannot be shared with all project administrators for your cluster."

What do you think? @Amrita42

Comment (Amrita42) [reply networking/multiple_networks/configuring-additional-network.adoc:32]: Sounds good , I have re-phrased as follows:
The NetworkAttachmentDefinition object is accessible by both project and cluster administrators. You should not store any sensitive information in this object if that information cannot be shared with the project administrators for your cluster. @ctauchen.

Comment (Amrita42) [reply networking/multiple_networks/configuring-additional-network.adoc:32]: let me re-phrase , thanks @ctauchen :+1:

Comment (ctauchen) [reply networking/multiple_networks/configuring-additional-network.adoc:32]: Super -- @zhaozhanqi does this still look good to you?

**Comment by phoracek** [comment]
/lgtm

Note that the PR this BZ is blocked on is still under review: https://bugzilla.redhat.com/show_bug.cgi?id=2021191

**Comment by Amrita42** [comment]
@phoracek Noted not to be merged till the related BZ is in a verified state.

**Comment by dougbtv** [comment]
Just throwing in a +1. Thanks for the docs update

File: networking/multiple_networks/configuring-additional-network.adoc (hunk 3 above)
Comment (zhaozhanqi) [reply networking/multiple_networks/configuring-additional-network.adoc:32]: yes, LGTM, sorry for late reply

**Pull request merged**